packages/components/psammead-social-embed/src/Canonical/index.jsx

```diff
@@ -59,6 +59,20 @@ export const providers = {
 const CanonicalEmbed = ({ provider, oEmbed }) => {
   useScript(providers[provider].script);
 
+  if (provider === 'twitter') {
```

Comment:
Do we need to worry about Facebook / YouTube / TikTok 🤣 ?

Reply:
We don't support Facebook and TikTok yet (notsureif joking 😉 ), youtube doesn't need any extra logic to work with lazy loading afaik.